docker-agent/gwc/cleancache.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+#
+################################################################################
+# This script allows to call the masstruncate functionality of GeoWebCache for 
+# all the layers that match specific regex for a cluster of GeoServer instances.
+# Usage: 
+# cleancache.sh regex
+# e.g. 
+# cleancache.sh "^topp\\:"
+# cleancache.sh "^topp\\:states$"
+# If regex not present use the standard input to get a list of layers to 
+# clear. Redirect a file or write the list and then type CTRL+D to start
+################################################################################
+#
+# author: Lorenzo Natali - lorenzo.natali@geo-solutions.it
+# 
+# Copyright 2015, GeoSolutions Sas.
+# All rights reserved.
+# 
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+# 
+################################################################################
+ 
+#GEOSERVER ADMINISTRATOR CREDENTIALS
+USER="admin"
+PASSWORD="geoserver"
+
+# Array of geoserver instances
+GEOSERVER_INSTANCES=("http://172.17.0.1:8080/master/gwc/rest/" "http://172.17.0.1:80/geoserver/gwc/rest/")
```

Comment:
let's talk about this.

Why not invoking the script two times instead? Can we still truncate the cache just for the master / just for the slaves or does it imply the same truncate operation for both?

Reply:
@randomorder that is how the script was defined (I'm not changing nothing more than list of instances).
So far, as I understand from code reading script is executing same call to rest api for the list of instances.